networktest.go

```diff
@@ -159,14 +161,27 @@ func (l *lightningNode) genArgs() []string {
 // start launches a new process running lnd. Additionally, the PID of the
 // launched process is saved in order to possibly kill the process forcibly
 // later.
-func (l *lightningNode) start() error {
+func (l *lightningNode) start(lndError chan error) error {
 	args := l.genArgs()
 
 	l.cmd = exec.Command("lnd", args...)
+
+	// Redirect stderr output to buffer
+	var errb bytes.Buffer
+	l.cmd.Stderr = &errb
+
 	if err := l.cmd.Start(); err != nil {
 		return err
 	}
 
+	go func() {
```

Comment:
Excellent addition! This should make debugging errors when writing integration tests a much, much more pleasant experience.